File: MPI/mpicheck/tests/functions/1.0/MPI_Intercomm_create.f90

        
        program main
        use mpi
        
        INTEGER var_0
       INTEGER var_1
       INTEGER var_2
       INTEGER var_3
       INTEGER var_4
       INTEGER var_5
       INTEGER var_6
        call mpi_intercomm_create(var_0, var_1, var_2, var_3, var_4, var_5, var_6)
       call pmpi_intercomm_create(var_0, var_1, var_2, var_3, var_4, var_5, var_6)
        end program main
    

File: MPI/mpicheck/tests/functions/1.0/MPI_Intercomm_create.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    MPI_Comm var_0;
    int var_1;
    MPI_Comm var_2;
    int var_3;
    int var_4;
    MPI_Comm *var_5;
    int ret;
    /* calls */
    ret = MPI_Intercomm_create(var_0, var_1, var_2, var_3, var_4, var_5);
    ret = PMPI_Intercomm_create(var_0, var_1, var_2, var_3, var_4, var_5);
    return 0;
}
